commit fcbecf2eda74f092e3b488506cdca9eb97cdd1e9
Author: Fabio M. Di Nitto <fdinitto@redhat.com>
Date:   Tue Sep 28 13:15:24 2010 +0200

    cman init: fix "stop remove" operation
    
    re-instate support for stop remove that was erroneusly dropped
    
    Resolves: rhbz#638954
    
    Signed-off-by: Fabio M. Di Nitto <fdinitto@redhat.com>

diff --git a/cman/init.d/cman.in b/cman/init.d/cman.in
index 787000f..50df96d 100644
--- a/cman/init.d/cman.in
+++ b/cman/init.d/cman.in
@@ -842,6 +842,9 @@ start)
 ;;
 stop)
 	cmanremove=""
+	if [ -n "$2" ] && [ "$2" = "remove" ]; then
+		cmanremove=remove
+	fi
 	stop && rm -f $LOCK_FILE
 	if [ "$INITLOGLEVEL" = "quiet" ]; then
 		success
